hailjwt/test/test_jwt.py

```diff
@@ -0,0 +1,12 @@
+import uuid
+import hailjwt as hj
+
+
+def test_round_trip():
```

Comment:
We should also verify rejection (1) corrupted key, and (2) corrupted cipher text.

Reply:
corrupted key means a key that's too short?

Reply:
I'd like to see tests that verify:
 - an attempt to modify the cypher text is detected,
 - if the cipher text was signed by another key (e.g. one generated by an attacker), it is detected.

Reply:
done